ambry-frontend/src/test/java/com.github.ambry.frontend/AmbryBlobStorageServiceTest.java

```diff
@@ -104,23 +121,74 @@
   private final FrontendTestResponseHandler responseHandler;
   private final InMemoryRouter router;
   private final ClusterMap clusterMap;
-
+  private final BlobId referenceBlobId;
+  private final String referenceBlobIdStr;
+  private final AccountServiceFactory accountServiceFactory;
+  private final VerifiableProperties verifiableProperties;
+  private final boolean shouldAllowServiceIdBasedPut;
   private AmbryBlobStorageService ambryBlobStorageService;
 
   /**
+   * Randomly generates a reference {@link Account} and {@link Container}s.
+   */
+  static {
+    short refAccountId = Utils.getRandomShort(random);
+    String refAccountName = UtilsTest.getRandomString(10);
+    Account.AccountStatus refAccountStatus =
+        random.nextBoolean() ? Account.AccountStatus.ACTIVE : Account.AccountStatus.INACTIVE;
+    short refContainerId = Utils.getRandomShort(random);
+    String refContainerName = UtilsTest.getRandomString(10);
+    Container.ContainerStatus refContainerStatus =
+        random.nextBoolean() ? Container.ContainerStatus.ACTIVE : Container.ContainerStatus.INACTIVE;
+    String refContainerDescription = UtilsTest.getRandomString(10);
+    boolean refContainerPrivacy = random.nextBoolean();
+    refContainer = new ContainerBuilder(refContainerId, refContainerName, refContainerStatus, refContainerDescription,
+        refContainerPrivacy, refAccountId).build();
+    refDefaultPublicContainer =
```

Comment:
We cannot, coz these containers are created per account. Their `parentAccountId`s are different.